stdlib/int64.gr

```diff
@@ -317,6 +584,20 @@ export let rec lor = (x: Int64, y: Int64) => {
   ret
 }
 
+/**
+ * @section Inspections: Functions for inspecting Int64 values.
+ */
```

Comment:
Not sure how I feel about this section. I don't really like the name.

It also seems to be in the wrong place because it is above `lxor`